include/midi.h

```diff
@@ -41,6 +41,33 @@ void MIDI_ListAll(Program *output_handler);
 void MIDI_RawOutByte(uint8_t data);
 void MIDI_ResumeSequence();
 
+enum class MessageType : uint8_t {
+	Channel,
+	Sysex,
+};
+
+struct MidiWork {
+	std::vector<uint8_t> message   = {};
+	uint16_t num_pending_frames    = 0;
```

Comment:
What does frame refer to here? Sample frames? Or something to do with [MIDI Time Code](https://en.wikipedia.org/wiki/MIDI_timecode) perhaps?

The commit message doesn't make it clear either as it seems to be derived from absolute time and/or "DOSBox tick" values?

"This number of frames is calculated between the prior rendering event time and the current message time, measured in DOS fractional ticks."

So at this point, I'm 100% clueless as to what a frame means here; I think this needs clarification.

Reply:
Didn't realize there were MIDI frames too - that would certainly make this confusing!

Renaming to `num_pending_audio_frames`; because that's the 1:1 type involved: `AudioFrame` that are pending ahead of this MIDI message.